.github/workflows/release-python.yml

```diff
@@ -0,0 +1,90 @@
+name: Release Python
+
+on:
+  push:
+    branches:
+      - 135-python-bindings-distribution
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
```

Comment:
Could you leave a comment here, for future-us, that describes
* why we don't include 3.7 here;
* why only 3.7 is built for x86_64; and
* why we don't include 3.7 for universal2.

I think I know the answer, but am likely to forget :D

Reply:
Done